velox/docs/ext/iceberg.py

```diff
@@ -0,0 +1,1101 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Would it make sense to introduce the iceberg document generation framework with another PR and focus on the functions support in this one?

Reply:
Do we need to include iceberg function document describing bucket function in this PR?